app/uploaders/asset_uploader.rb

```diff
@@ -88,6 +88,16 @@ class AssetUploader < Kithe::AssetUploader
   end
 
 
+  # For FLAC originals, let's create a mono m4a derivative.
+  Attacher.define_derivative('m4a', content_type: "audio") do |original_file, attacher:|
+    # Both audio/flac or audio/x-flac seem to be valid
+    if attacher.file.content_type.match /flac$/
+      Kithe::FfmpegTransformer.new(
+        bitrate: '64k', force_mono: true, audio_codec: 'aac', output_suffix: 'm4a',
+      ).call(original_file)
```

Comment:
maybe mention in comment that the output characteristics, including relatively low 64K bitrate, is based on assumption these are Oral History voice recordings. And maybe mention the combined audio derivative, which has matching choices.

Reply:
Yeah -- haven't put much thought into the bitrate; for now, just copied the settings from the combined audio derivs.

Reply:
I think they are probably right -- I think I remmeber you DID put a lot of thought into em when we made the combined audio derivs!  I don't know that it needs to be re-visited. 

I just think they would not necessarily be appropriate for musical instead of voice content -- which we don't _expect_ to have... The combined audio derivs actually *only functions* on Oral Histories. But this new feature will function on any FLAC uploaded. We don't have any plans for anything but OH... I was just thinking leaving a comment explaining where the settings came (copied from combined audio derivs, if changed in one place should be changed in both) is a good idea at this point.

Reply:
Definitely - music recorded in stereo would be pretty much ruined in this format. These are mono and the lowest possible quality that's acceptable for listening to a conversation between two people. The use case is "I want to download these quickly and listen to them".